flake.nix

```diff
@@ -1,7 +1,7 @@
 {
   description = "Hydra's builtin hydra-eval-jobs as a standalone";
 
-  inputs.nixpkgs.url = "https://nixos.org/channels/nixpkgs-unstable/nixexprs.tar.xz";
+  inputs.nixpkgs.url = "https://nixos.org/channels/nixos-25.11/nixexprs.tar.xz";
```

Comment:
This change was required because we need https://github.com/NixOS/nixpkgs/pull/462747 to be merged to master, but it was only merged to staging and various release branches. I have requested that it be merged to master as well: https://github.com/NixOS/nixpkgs/pull/462747#issuecomment-3667073278